app/services/foreman/plugin.rb

```diff
@@ -396,5 +405,30 @@ def extend_template_helpers_by_module(mod)
       end
       allowed_template_helpers(*(mod.public_instance_methods - Module.public_instance_methods))
     end
+
+    def add_manager_roles(resources, opts)
+      manager = Role.find_by :name => 'Manager'
```

Comment:
what happens if user deleted the role? should we recreate it? also I need to test it but I think this can happen when you install Foreman, DB was not seeded yet but the rails env loads which means it loads all plugin initializers.